ansible/roles/ocp4-workload-quarkus-super-heroes-demo/.yamllint

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
This should be removed. If a line will break yamllint it needs to be fixed - single line exceptions are acceptable. But not a blanket change.

Reply:
This has been removed.